etna/search/tests/test_forms.py

```diff
@@ -41,116 +41,60 @@ def test_empty_form(self):
         form.is_valid()
 
         self.assertEqual(
-            form.selected_filters(),
-            {
-                "levels": [],
-                "topics": [],
-                "collections": [],
-                "closure_statuses": [],
-                "catalogue_sources": [],
-            },
-        )
-
-    def test_selected_group_is_excluded(self):
-        form = CatalogueSearchForm(
-            {
-                "group": "group:tna",
-            }
-        )
-
-        form.is_valid()
-
-        self.assertEqual(
-            form.selected_filters(),
-            {
-                "levels": [],
-                "topics": [],
-                "collections": [],
-                "closure_statuses": [],
-                "catalogue_sources": [],
-            },
+            form.selected_filters,
+            {},
         )
 
-    def test_selected_level(self):
+    def test_with_recognised_filter_values(self):
         form = CatalogueSearchForm(
             {
                 "group": "group:tna",
-                "levels": ["level:level-one"],
+                "level": ["Division"],
+                "collection": ["WO"],
```

Comment:
Can we have the other bits from the selected list as well - _topic, closure, catalogue_source_ , since those test cases are removed?

Reply:
Those other fields don't apply any validation, so including them in a validation test is just noise that we do not need really.

Reply:
Have included separate valid/partially invalid/completely invalid form tests now, with a mix of choice-configured and non choice-configured fields where possible.